src/components/AuthPage/ForgotPassword/index.js

```diff
@@ -26,15 +26,25 @@ const ForgotPassword = ({
   const [error, setError] = useState("");
   const [success, setSuccess] = useState(false);
   const [email, setEmail] = useState("");
+  const [conform, conformemail] = useState(true);
   const [open, setOpen] = React.useState(true);
-  const errorProps = useSelector(({ auth }) => auth.profile.error);
+  const errorProps = useSelector(({ auth }) => {
+  console.log(auth)
+  return auth.profile.error
+});
   const loadingProps = useSelector(({ auth }) => auth.profile.loading);
   const dispatch = useDispatch();
 
   useEffect(() => setError(errorProps), [errorProps]);
   useEffect(() => setLoading(loadingProps), [loadingProps]);
   useEffect(() => setOpen(true), [loadingProps]);
-
+  useEffect(()=>{
+    if(email.includes('@gmail.com') && /[0-9]/.test(email) && /\p{L}/u.test(email)){
```

Comment:
what about the emails containing "yahoo" or other such email services ?, I guess that would be a problem if we set it only for gmail

Reply:
but in codelabz website it only have gmail option.

Reply:
Thanks @ABHISHEK-PANDEY2 for pointing out!

@kushalShukla-web the email - password sign in can have other emails too,
you can use something like `^[\w-\.]+@([\w-]+\.)+[\w-]{2,4}$` to verify the entire email at once!